Sources/NIOCore/ByteBuffer-core.swift

```diff
@@ -169,6 +198,11 @@ public struct ByteBufferAllocator: Sendable {
 ///
 /// If needed, `ByteBuffer` will automatically resize its storage to accommodate your `set` request.
 ///
+/// > Note: If you need to access the contents of a `ByteBuffer` with `get*(at: index)` methods, make sure that the
```

Comment:
I think the DocC magic you're looking for here, instead of Markdown quotation, is:

```swift
/// - Note: ...
/// ...
/// ...
```

Reply:
You're right! Both work, but `- Note:` is cleaner.